Generator/Templates/Shared/struct.sbntxt

```diff
@@ -7,28 +7,25 @@ namespace {{ namespace }}
 {
     // AUTOGENERATED FILE - DO NOT MODIFY
 
-{{
-include 'helper.sbntxt'
-this | write_summary '    '
-this | write_obsolete_attribute '    '
-~}}
+    {{ include 'header.sbntxt' this ~}}
     [StructLayout(LayoutKind.Sequential)]
```

Comment:
This line get one more indentation level (4 spaces) in the generated code

Reply:
This works for me e.g.:

```C#
/// </summary>
[StructLayout(LayoutKind.Sequential)]
public partial struct FlagsClass
{
```

Reply:
This issue also appear only when the struct doesn't have a doc comment (like the AboutDialogClass struct)